consensus/src/alerts/service.rs

```diff
@@ -20,6 +20,14 @@ pub struct Service<H: Hasher, D: Data, MK: MultiKeychain> {
     rmc: ReliableMulticast<H::Hash, MK>,
     messages_for_rmc: Sender<RmcMessage<H::Hash, MK::Signature, MK::PartialMultisignature>>,
     messages_from_rmc: Receiver<RmcMessage<H::Hash, MK::Signature, MK::PartialMultisignature>>,
+
+    data_for_backup: Sender<AlertData<H, D, MK>>,
+    responses_from_backup: Receiver<AlertData<H, D, MK>>,
+
+    own_alert_responses: HashMap<H::Hash, OnOwnAlertResponse<H, D, MK>>,
+    network_alert_responses: HashMap<H::Hash, OnNetworkAlertResponse<H, D, MK>>,
+    multisigned_notifications: HashMap<H::Hash, ForkingNotification<H, D, MK::Signature>>,
```

Comment:
I would sleep better if we would change hashmaps to some bounded-size version of it.

Reply:
Hm, good point, some `LruCache` or something probably...

Reply:
done

Reply:
Cool, I have no idea about limits here. @timorl pls resolve this conversation if you find 1024 reasonable limit :)

Reply:
Whoops, turns out this was a bad idea after all, sorry. ._.''